src/pages/keyword-generator/KeywordGenerator.vue

```diff
@@ -26,33 +26,38 @@
             <template #title>Exclude Words</template>
           </ma-input>
           <div class="multi-select-container">
```

Comment:
For some parts we used the tailwind classes directly, for other parts we used the `@apply` property. For consistency we should choose one of the two methods.